.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,106 @@
+name: Bump Version
+
+on:
+  workflow_dispatch:
+    inputs:
+      isMajor:
+        description: 'Major Update'
+        required: true
+        default: 'false'
+      isMinor:
+        description: 'Minor Update'
+        required: true
+        default: 'false'
+      newVersion: 
+        description: 'New Version'
+        required: false
+
+jobs:
+  bump-version:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Check Acccess
+        uses: scoremedia/action-repository-permission@v1
+        with:
+          required-permission: admin
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: master
+
+      - id: update-version
+        name: Update version
+        working-directory: Sources
+        run: |
+          version_row="$(find . -type f -name "*.podspec" -print0 | xargs -0 grep -o "s.version          = .*")"
```

Comment:
Do the spaces have to be accurate? Seems like it can cause headaches in the future

Reply:
Hmm I think so. Let me tinker with the regex a little.

Reply:
I updated it but can't do anything for the replacement string

Reply:
That's ok, it's a lot cleaner now. Thanks!